CDC-Data-Reconciliation-Backend/auto-compare.py

```diff
@@ -0,0 +1,47 @@
+import csv
+import argparse
+from datetime import datetime
+import os
+import pyodbc
+from fastapi import FastAPI, File, Response, UploadFile, HTTPException
+from fastapi.middleware.cors import CORSMiddleware
+import json
+
+app = FastAPI()
+
+origins = [
+    "http://localhost:5173"
+]
+
+app.add_middleware(CORSMiddleware, allow_origins=origins,
+                   allow_credentials=True, allow_methods=["*"], allow_headers=["*"])
+
+
+# Method to test if you have the correct ODBC Driver
+# print("List of ODBC Drivers:")
+# dlist = pyodbc.drivers()
+# for drvr in dlist:
+#     print('LIST OF DRIVERS:' + drvr)
+
+# Load config.json
+app.dir = os.path.dirname(__file__)
+
+
+config_file_path = os.path.join(app.dir, "config.json")
+with open(config_file_path, "r") as f:
+    app.config = json.load(f)
+
+def main():
```

Comment:
The main method is never called... Put 
`if __name__ == "__main__": 
    main()
`
at the bottom of your code